.github/workflows/ci.yml

```diff
@@ -270,6 +270,39 @@ jobs:
 
           _make PLATFORM=virt
 
+  QEMUv7_check:
+    name: make check (QEMUv7)
+    runs-on: ubuntu-latest
+    container: jforissier/optee_os_ci:qemu_check
```

Comment:
Not a need for this PR and right now, but would it be possible to change jforissier to a Linaro owned thing controlled by you and a few others?

Reply:
Sure. There are two things to consider: the source code: https://github.com/jforissier/docker_optee_os_ci/tree/qemu_check, and the Docker Hub repository that stores the compiled images: https://hub.docker.com/r/jforissier/optee_os_ci.